extern/tbb/CMakeLists.txt

```
@@ -27,5 +27,13 @@ FetchContent_Declare( TBB ${TRACCC_TBB_SOURCE} )
 set( TBB_TEST FALSE CACHE BOOL "Turn off the TBB tests" )
 set( TBB_STRICT FALSE CACHE BOOL "Do not throw errors on compiler warnings" )
 
+# Treat the TBB headers as "system headers", to avoid getting warnings from
+# them.
+get_target_property( _incDirs tbb INTERFACE_INCLUDE_DIRECTORIES )
+target_include_directories( tbb
+   SYSTEM INTERFACE ${_incDirs} )
+unset( _incDirs )
```

:thinking: TBB is actually a `SHARED` library, not an `INTERFACE` one. (I didn't think about this in the office.) So you'll likely rather need to do:

```suggestion
get_target_property( _incDirs tbb INCLUDE_DIRECTORIES )
target_include_directories( tbb
   SYSTEM PUBLIC ${_incDirs} )
unset( _incDirs )
```